src/app/app.component.html

```diff
@@ -1,2 +1 @@
-
-<router-outlet></router-outlet>
+<router-outlet></router-outlet>
```

Comment:
Please add selector to for login page. Without extra knowledge and work i put in i can't see what you actually worked on . I know you will have to delete is later on but put up with it. A thousand mile road is still done step by step. :D

Reply:
Fixed